### PR TITLE
Add versioning for the client

### PR DESCRIPTION
**Pull request by OleksandrOrlov**
**NOTE: This PR is dependent on https://github.com/libyui/libyui-rest-api/pull/4 will be merged. So, please do not merge it until the PR will be merged.**

LibYUI Rest API project introduced versioning. The commit adds the
appropriate versioning for the client, so that client will be compatible
with the respective API.

Related ticket: https://progress.opensuse.org/issues/62531

### CODE REVIEW

File: lib/libyui_client/process_helper.rb [hunk 1]
```diff
@@ -3,19 +3,28 @@
...
   # default timeout for process
   DEFAULT_TIMEOUT_PROCESS = 20
 
+  def self.set_host
+    ENV['YUI_HTTP_HOST'] ||= read_config_value('YUI_HTTP_HOST')
+  end
+
   # set the application introspection port for communication
   def self.set_port
     # 14155 is currently an unassigned port
```
Comment (jknphy) [inline lib/libyui_client/process_helper.rb]: let's remove this comment then

File: lib/libyui_client/process_helper.rb [hunk 2]
```diff
@@ -3,19 +3,28 @@
...
 require 'socket'
 require 'rainbow'
+require 'libyui_client/config_reader'
 
 # Client to interact with YAST UI rest api framework for integration testing
 module LibyuiClient
   # default timeout for process
   DEFAULT_TIMEOUT_PROCESS = 20
 
+  def self.set_host
```
Comment (jknphy) [inline lib/libyui_client/process_helper.rb]: Perhaps a comment here

File: lib/libyui_client/process_helper.rb [hunk 3]
```diff
@@ -3,19 +3,28 @@
...
+
   # set the application introspection port for communication
   def self.set_port
     # 14155 is currently an unassigned port
-    ENV['YUI_HTTP_PORT'] ||= '14155'
+    ENV['YUI_HTTP_PORT'] ||= read_config_value('YUI_HTTP_PORT')
     ENV['YUI_HTTP_PORT']
   end
 
+  def self.set_api_version
```
Comment (jknphy) [inline lib/libyui_client/process_helper.rb]: a another one here would not hurt :) But we can also do it later, I see that in other files I missed some comments too :) just to consistent with what is already in the file but not an important thing at the moment.

**Comment by OleksandrOrlov** [comment]
Closed, as the client structure is changed and currently version should be added in another way.

**Pull request closed without merging**